packages/gtk4.rb

```diff
@@ -3,25 +3,23 @@
 class Gtk4 < Package
   description 'GTK+ is a multi-platform toolkit for creating graphical user interfaces.'
   homepage 'https://developer.gnome.org/gtk4/'
-  @_ver = '4.2.1'
+  @_ver = '4.3.1'
   @_ver_prelastdot = @_ver.rpartition('.')[0]
   version @_ver
   license 'LGPL-2.1'
-  compatibility 'all'
+  compatibility 'x86_64 aarch64 armv7l'
   source_url 'https://gitlab.gnome.org/GNOME/gtk.git'
   git_hashtag @_ver
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gtk4/4.2.1_armv7l/gtk4-4.2.1-chromeos-armv7l.tpxz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gtk4/4.2.1_armv7l/gtk4-4.2.1-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gtk4/4.2.1_i686/gtk4-4.2.1-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gtk4/4.2.1_x86_64/gtk4-4.2.1-chromeos-x86_64.tpxz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gtk4/4.3.1_armv7l/gtk4-4.3.1-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gtk4/4.3.1_armv7l/gtk4-4.3.1-chromeos-armv7l.tpxz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gtk4/4.3.1_x86_64/gtk4-4.3.1-chromeos-x86_64.tpxz'
   })
   binary_sha256({
-    aarch64: '131b1ef2ee52e9e4e19ac99a2973f2a6b0322d9bb02b7141f792bfe9f6f0b7e8',
-     armv7l: '131b1ef2ee52e9e4e19ac99a2973f2a6b0322d9bb02b7141f792bfe9f6f0b7e8',
-       i686: '49d26942d98c39a093fd341cfb5f62c97d869c20246304ccd8f9ff1cee5c4b92',
-     x86_64: '47520c1d162a3f626e7df1dd50a936b737ce4cf09e76ed1cec228e806fc3e321'
+    aarch64: 'd714a0b5c0318ee50ad3e7bd581e4d69435aafcf3800b640030f940e5accb1cd',
+     armv7l: 'd714a0b5c0318ee50ad3e7bd581e4d69435aafcf3800b640030f940e5accb1cd',
+     x86_64: '986e9290a4ba1e216db1f7d6c89ce4d86cf9ecae1af3b90c1ad43fd385237c75'
   })
```

Comment:
If we are going to remove i686, we need to also remove i686 binaries for all packages that depend on gtk4 also, otherwise kludgy errors will occur.

Reply:
good call. Direct dependencies on gtk4 have had i686 binaries removed.